RevenueCatUI/Views/FooterView.swift

```diff
@@ -0,0 +1,174 @@
+//
+//  FooterView.swift
+//  
+//
+//  Created by Nacho Soto on 7/20/23.
+//
+
+import RevenueCat
+import SwiftUI
+
+@available(iOS 16.0, macOS 13.0, tvOS 16.0, *)
+struct FooterView: View {
+
+    var configuration: PaywallData.Configuration
+    var colors: PaywallData.Configuration.Colors
+    var purchaseHandler: PurchaseHandler
+
+    var body: some View {
+        HStack {
+            if self.configuration.displayRestorePurchases {
+                RestorePurchasesButton(purchaseHandler: self.purchaseHandler)
+
+                self.separator
+                    .hidden(if: !self.hasTOS && !self.hasPrivacy)
+            }
+
+            if let url = self.configuration.termsOfServiceURL {
+                LinkButton(
+                    url: url,
+                    // Fix-me: localize
+                    titles: "Terms and conditions", "Terms"
+                )
+
+                self.separator
+                    .hidden(if: !self.hasPrivacy)
+            }
+
+            if let url = self.configuration.privacyURL {
+                LinkButton(
+                    url: url,
+                    // Fix-me: localize
+                    titles: "Privacy policy", "Privacy"
+                )
+            }
+        }
+        .foregroundColor(self.colors.foregroundColor)
+        .font(.caption.bold())
+        .padding(.horizontal)
+    }
+
+    private var separator: some View {
+        Image(systemName: "circle.fill")
+            .font(.system(size: 5))
+    }
+
+    private var hasTOS: Bool { self.configuration.termsOfServiceURL != nil }
+    private var hasPrivacy: Bool { self.configuration.privacyURL != nil }
+
+}
+
+@available(iOS 16.0, macOS 13.0, tvOS 16.0, *)
+private struct RestorePurchasesButton: View {
+
+    let purchaseHandler: PurchaseHandler
+
+    @State
+    private var restored = false
+
+    var body: some View {
+        AsyncButton {
+            _ = try await self.purchaseHandler.restorePurchases()
+            self.restored = true
+        } label: {
+            // Fix-me: localize
+            ViewThatFits {
+                Text("Restore Purchases")
```

Comment:
Should this be `Restore purchases` to match the casing in `Privacy policy` and `Terms and conditions`?

```suggestion
                Text("Restore purchases")
```

Reply:
Agreed 👍🏻